src/kem/ntru/pqclean_ntruhrss701_avx2/poly_s3_to_rq.s

```diff
@@ -0,0 +1,3118 @@
+vpor %ymm3, %ymm2, %ymm3
+vmovdqa %ymm3, 1248(%rsp)
+vpaddw 1280(%rsp), %ymm1, %ymm2
+vpsrlw $8, %ymm2, %ymm3
+vpand mask_ff(%rip), %ymm2, %ymm2
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_f(%rip), %ymm3, %ymm2
+vpsrlw $4, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpsubw mask_3(%rip), %ymm3, %ymm14
+vpsraw $15, %ymm14, %ymm15
+vpandn %ymm14, %ymm15, %ymm2
+vpand %ymm15, %ymm3, %ymm14
+vpxor %ymm14, %ymm2, %ymm3
+vpsrlq $1, %ymm3, %ymm2
+vpsubw %ymm2, %ymm0, %ymm2
+vpand const_modq(%rip), %ymm2, %ymm2
+vpand const_1s(%rip), %ymm3, %ymm3
+vpor %ymm3, %ymm2, %ymm3
+vmovdqa %ymm3, 1280(%rsp)
+vpaddw 1312(%rsp), %ymm1, %ymm2
+vpsrlw $8, %ymm2, %ymm3
+vpand mask_ff(%rip), %ymm2, %ymm2
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_f(%rip), %ymm3, %ymm2
+vpsrlw $4, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpsubw mask_3(%rip), %ymm3, %ymm14
+vpsraw $15, %ymm14, %ymm15
+vpandn %ymm14, %ymm15, %ymm2
+vpand %ymm15, %ymm3, %ymm14
+vpxor %ymm14, %ymm2, %ymm3
+vpsrlq $1, %ymm3, %ymm2
+vpsubw %ymm2, %ymm0, %ymm2
+vpand const_modq(%rip), %ymm2, %ymm2
+vpand const_1s(%rip), %ymm3, %ymm3
+vpor %ymm3, %ymm2, %ymm3
+vmovdqa %ymm3, 1312(%rsp)
+vpaddw 1344(%rsp), %ymm1, %ymm2
+vpsrlw $8, %ymm2, %ymm3
+vpand mask_ff(%rip), %ymm2, %ymm2
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_f(%rip), %ymm3, %ymm2
+vpsrlw $4, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpsubw mask_3(%rip), %ymm3, %ymm14
+vpsraw $15, %ymm14, %ymm15
+vpandn %ymm14, %ymm15, %ymm2
+vpand %ymm15, %ymm3, %ymm14
+vpxor %ymm14, %ymm2, %ymm3
+vpsrlq $1, %ymm3, %ymm2
+vpsubw %ymm2, %ymm0, %ymm2
+vpand const_modq(%rip), %ymm2, %ymm2
+vpand const_1s(%rip), %ymm3, %ymm3
+vpor %ymm3, %ymm2, %ymm3
+vmovdqa %ymm3, 1344(%rsp)
+vpaddw 1376(%rsp), %ymm1, %ymm2
+vpsrlw $8, %ymm2, %ymm3
+vpand mask_ff(%rip), %ymm2, %ymm2
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_f(%rip), %ymm3, %ymm2
+vpsrlw $4, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpand mask_3(%rip), %ymm3, %ymm2
+vpsrlw $2, %ymm3, %ymm3
+vpaddw %ymm3, %ymm2, %ymm3
+vpsubw mask_3(%rip), %ymm3, %ymm14
+vpsraw $15, %ymm14, %ymm15
+vpandn %ymm14, %ymm15, %ymm2
+vpand %ymm15, %ymm3, %ymm14
+vpxor %ymm14, %ymm2, %ymm3
+vpsrlq $1, %ymm3, %ymm2
+vpsubw %ymm2, %ymm0, %ymm2
+vpand const_modq(%rip), %ymm2, %ymm2
+vpand const_1s(%rip), %ymm3, %ymm3
+vpor %ymm3, %ymm2, %ymm3
+vmovdqa %ymm3, 1376(%rsp)
+mov %rsp, %rsi
+call PQCLEAN_NTRUHRSS701_AVX2_poly_Rq_mul_x_minus_1
```

Comment:
Changing this to
`call PQCLEAN_NTRUHRSS701_AVX2_poly_Rq_mul_x_minus_1@plt`
should fix the PIC issue.

Reply:
Yes, it seems to fix it for me on the gcc7-shared build. Can you make the commit at PQClean and then I'll pull it from there?